conf/share-smb.conf

```
@@ -4,7 +4,7 @@
    path = /home/yunohost.app/samba/share
    guest ok = no
    browsable = yes
-   valid users = @samba.share
+   valid users = @sambashare
```

```suggestion
   valid users = @samba.share
```

We want the permission group Samba (share) to be added to the user authorized. So the dot is important.

---

Fine, but when doing some tests, having `@samba.share` failed ...